1080i/SkinSettings.xml

```diff
@@ -274,6 +274,7 @@
                         <label>$LOCALIZE[31239]</label>
                         <label2>$VAR[Label_BusyWaves]</label2>
                         <onclick>Skin.ToggleSetting(DisableBusyBackground)</onclick>
+                        <onclick condition="!System.HasAddon(resource.images.arctic.waves)">InstallAddon(resource.images.arctic.waves)</onclick>
```

Comment:
All I'm not certain on here is if there's a way to make this only popup the dialog when turning on the "Waves" animation. Currently, if the setting is on "None", a single click prompts for install, but declining still subsequently sets the setting, and attempting to change it back to "None" prompts for install again.

If it isn't already, it's probably a good idea to disable the "Waves" animation by default now, but I'm not sure where that is done in the skin files.